crates/bevy_pbr/src/lib.rs

```diff
@@ -1,6 +1,11 @@
 #![expect(missing_docs, reason = "Not all docs are written yet, see #3492.")]
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
-#![deny(unsafe_code)]
+#![forbid(unsafe_code)]
```

Comment:
I searched through `bevy_pbr`, and found no unsafe code - so I figured this was meant to be a `forbid`.